Java/Tests/jdi-uitest-webtests/src/main/resources/driver/temp/LATEST_RELEASE.txt

```diff
@@ -1 +0,0 @@
-2.31
```

Comment:
You can see it on other branches - it is there (may be got deleted with the build)